.travis.yml

```diff
@@ -51,19 +61,22 @@ cache:
    - $HOME/downloads
    - $HOME/.cache/pip
    - $HOME/Library/Caches/pip
+   - $HOME/Library/Caches/Homebrew
+before_cache:
+  - if [ "${TRAVIS_OS_NAME}" = "osx" ]; then brew cleanup; fi
 install:
   - mkdir -p "$HOME/downloads"
   - mkdir -p "$TRAVIS_BUILD_DIR/deps/cache/"
   - find "$HOME/downloads" -type f -exec cp -v {} "$TRAVIS_BUILD_DIR/deps/cache/" \;
-  - on_host ./install.sh --develop --python=python3 --with-qt
+  - on_host virtualenv --python=python3 jmvenv
   - on_host find "$TRAVIS_BUILD_DIR/deps/cache/" -type f -exec cp -v {} "$HOME/downloads/" \;
 before_script:
   - on_host source jmvenv/bin/activate
 script:
-  - on_host do_on linux bitcoind --help | head -1
-  - on_host do_on linux ./test/run_tests.sh
-  - on_host do_on linux flake8 jmbase jmbitcoin jmclient jmdaemon scripts test
+  - on_host bitcoind --help | head -1
+  - on_host ./test/run_tests.sh
+  - on_host flake8 jmbase jmbitcoin jmclient jmdaemon scripts test
```

Comment:
Maybe we should limit the linter to only run on linux?

Reply:
Ok.